modules/openapi-generator/src/main/resources/JavaSpring/converter.mustache

```diff
@@ -0,0 +1,27 @@
+package org.openapitools.configuration;
+
+import {{modelPackage}}.*;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.convert.converter.Converter;
+
+@Configuration
```

Comment:
This will create an empty configuration file if there are no enums. Might be confusing to users who generate the project and then don't understand why this file exists - maybe conditionally add this file if there are any enums present?

Reply:
I've added some code to check if there is an enum in the models.